tests/phpunit/tests/theme/wpTheme.php

```diff
@@ -375,6 +375,33 @@ public function test_get_files_nonexistent_theme() {
 		$this->assertEmpty( $files );
 	}
 
+	/**
+	 * Test wp_customize_url with existing query args.
+	 *
+	 * @ticket 63632
+	 */
+	public function test_wp_customize_url_with_existing_query_args() {
```

Comment:
There seem to be two separate test cases in this test. You might want to split it up into multiple tests, one for each assertion. In this way, the `admin_url` filter for one test assertion won't be applying for the second test assertion as well.

Reply:
I split the tests and added another for array query args, just to make sure multiple "theme[]=" would be removed.